internal/api/group.go

```diff
@@ -21,6 +21,54 @@ func initGroup(apiRouter *mux.Router, context *Context) {
 	groupRouter.Handle("", addContext(handleGetGroup)).Methods("GET")
 	groupRouter.Handle("", addContext(handleUpdateGroup)).Methods("PUT")
 	groupRouter.Handle("", addContext(handleDeleteGroup)).Methods("DELETE")
+
+	groupRouter.Handle("/mattermost/{version:[A-Za-z0-9.]+}", addContext(handleChangeMattermostVersion)).Methods("PUT")
+}
+
+// handleChangeMattermostVersion responds to PUT
+// /api/group/{group}/mattermost/{version}, upgrading or downgrading
+// the installation to the Mattermost version embedded in the request.
+func handleChangeMattermostVersion(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: `handleChangeMattermostVersion` is a really generic name when we probably want something more like `handleChangeGroupMattermostVersion`

Reply:
I think that's a great suggestion and actually I don't think it's a nitpick either, names are important!